test/index.js

```diff
@@ -877,40 +877,48 @@ describe('normalizr', function () {
     });
   });
 
-  it('can normalize nested entities', function () {
```

Comment:
Please don't change this test, just add another. Right now, the tests also act as documentation for what can be done.